zone/zonedb.h

```diff
@@ -451,6 +451,8 @@ class ZoneDatabase : public SharedDatabase {
 	bool		SetSpecialAttkFlag(uint8 id, const char* flag);
 	bool		GetPetEntry(const char *pet_type, PetRecord *into);
 	bool		GetPoweredPetEntry(const char *pet_type, int16 petpower, PetRecord *into);
+	std::string GetCustomPetName(uint32 char_id);
```

Comment:
Formatting is off here

Reply:
Hrmm, whole files formatting is messed up/inconsistent.
[https://github.com/EQEmu/Server/blob/master/zone/zonedb.h#L426](https://github.com/EQEmu/Server/blob/master/zone/zonedb.h#L426)

Ironically i see my edit like:
![image](https://user-images.githubusercontent.com/10248701/76576552-839cd200-6516-11ea-8f17-d130cfb0c7a3.png)